dance/modules/multi_modality/match_modality/scmm.py

```diff
@@ -194,8 +195,10 @@ def read_count(self, x):
         return (read)
 
     def forward(self, x):
+        x = torch.nan_to_num(x)
```

Comment:
Please comment on why this is needed (and the one below). To prevent under-/over-flow?

Reply:
I need to think about it. At first I thought it was a problem with the filtering of the original data set, but now when I use the filtered data set, nan still appears.

Reply:
After filtering data, I have deleted it